README.md

```diff
@@ -157,13 +150,13 @@ var awesomeapp = new off.Offliner('myawesomeapp.com');
 var terrificapp = new off.Offliner('myterrificapp.com');
 ```
 
-## Using offliner as serviceworkerware middleware
+#### Using offliner as serviceworkerware middleware
```

Comment:
Can we add this is experimental? As serviceworkerware is evolving quickly we can not guarantee the compatibility at the moment.